docs/sources/features/datasources/influxdb.md

```diff
@@ -203,24 +202,21 @@ be applied to all your InfluxDB queries.
 
 ## Annotations
 
-[Annotations]({{< relref "reference/annotations.md" >}}) allows you to overlay rich event information on top of graphs. You add annotation
-queries via the Dashboard menu / Annotations view.
+[Annotations]({{< relref "reference/annotations.md" >}}) allows you to overlay rich event information on top of graphs. Add annotation queries via the Dashboard menu / Annotations view.
 
 An example query:
 
 ```SQL
 SELECT title, description from events WHERE $timeFilter ORDER BY time ASC
 ```
 
-For InfluxDB you need to enter a query like in the above example. You need to have the ```where $timeFilter```
-part. If you only select one column you will not need to enter anything in the column mapping fields. The
-Tags field can be a comma separated string.
+For InfluxDB, you need to enter a query like the one in the example above. The ```where $timeFilter``` component is required. If you only select one column you need not enter anything in the column mapping fields. The Tags field can be a comma separated string.
```

Comment:
```suggestion
For InfluxDB, you need to enter a query like the one in the example above. The ```where $timeFilter``` component is required. If you only select one column, then you do not need to enter anything in the column mapping fields. The **Tags** field can be a comma-separated string.
```

Reply:
@marcusolsson just suggested the "need not" wording. I don't feel strongly either way.